### PR TITLE
Feature/test user tools

### PR DESCRIPTION
**Pull request by mattash**
This adds a new module for the app called "app_management" that can be used to post changelog updates for users to learn more about what changes are being pushed to the app. 

It also adds a button for users to provide feedback and submit bug reports.

### CODE REVIEW

File: hub/templates/changelog.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "base_generic.html" %}
+
+{% block content %}
+<div class="row w-100 p-3 mx-3 mb-3 rounded-bottom shadow bg-light text-dark">
+
+  <h2>Changelog Updates</h2>
+  {% for changelog in changelogs %}
```
Comment (andylitalo) [inline hub/templates/changelog.html:7]: In the edge case where there are no changelogs, this loop throws an error. Should we just skip the loop with an `if` block if there are no changelogs, or is the error good as a reminder to populate the changelog field?

**Review by andylitalo — CHANGES_REQUESTED** [review]
One small question that might need a code change for displaying changelogs when there is none in the database

File: hub/templates/changelog.html (hunk 1 above)
Comment (mattash) [reply hub/templates/changelog.html:7]: Good point. I can handle that.

Comment (mattash) [reply hub/templates/changelog.html]: fixed here: https://github.com/surp-hovhannes/bahk/pull/32/commits/dfe3ad7e055437f2027c97288fc0e74828a095da

**Review by andylitalo — APPROVED** [review]
LGTM!

**Pull request merged**